pytorch_lightning/loggers/neptune.py

```diff
@@ -17,23 +17,34 @@
 """
 import logging
 from argparse import Namespace
-from typing import Any, Dict, Iterable, Optional, Union
+from typing import Any, Dict, Optional, Union
 
 import torch
-from torch import is_tensor
 
 from pytorch_lightning.loggers.base import LightningLoggerBase, rank_zero_experiment
 from pytorch_lightning.utilities import _module_available, rank_zero_only
 
 log = logging.getLogger(__name__)
-_NEPTUNE_AVAILABLE = _module_available("neptune")
+
+if _module_available("neptune"):
+    from neptune import __version__
+
+    _NEPTUNE_AVAILABLE = __version__.startswith('0.9.') or __version__.startswith('1.')
+else:
+    _NEPTUNE_AVAILABLE = False
```

Comment:
Can you do something like this instead:

```
_NEPTUNE_AVAILABLE = _module_available("neptune")
_NEPTUNE_GREATER_EQUAL_0_9 = _NEPTUNE_AVAILABLE and _compare_version("neptune", operator.ge, "0.9.0")
```

And then use these properties for imports instead of `try-except`?

Reply:
yeah version would be better as you never now when the new API become also Legacy :]

Reply:
ok